roles/cifmw_cephadm/tasks/logs.yml

```diff
@@ -0,0 +1,41 @@
+- name: Get ceph_cli
+  ansible.builtin.include_tasks: ceph_cli.yml
+
+- name: Get the ceph orchestrator status
+  ansible.builtin.command: "{{ cifmw_cephadm_ceph_cli }} orch status --format json"
+  register: ceph_orch_status
+  become: true
+
+- name: Fail if ceph orchestrator is not available
+  ansible.builtin.fail:
+    msg: "'ceph orch status' returned {{ ceph_orch_status.stdout | from_json }}"
+  when:
+    - not (ceph_orch_status.stdout | from_json).available
+
+- name: Create ceph-logs directory
+  when: cifmw_cephadm_dump | default(false)
+  ansible.builtin.file:
+    path: "{{ ansible_user_dir }}/ci-framework-data/ceph-logs"
+    state: directory
+    mode: "0755"
+
+- name: Show the status of the resulting deployed Ceph cluster
+  become: true
+  vars:
+    ceph_log_path: "{{ ansible_user_dir }}/ci-framework-data/ceph-logs/ceph_{{ cmd_type }}.log"
```

Comment:
@fultonj @rebtoor The only potential thing the might result problematic is this path (I'm wondering if we need extra changes to make sure it's collected by the job).
Let's see the first CI run and eventually update the patch.